test/testProject.js

```diff
@@ -125,6 +138,102 @@ module.exports.project = {
         test.done();
     },
     
+    testProjectGeneratesCorrectPluralCategoriesInNewStringsXliffs: function(test){
+        test.expect(9);
```

Comment:
This feature is only available xliff 1.2 version. isn't it? If so, I think it needs to write the exact xliff working version in the Release note.  or If it's work xliff 2.0 too, test case should be added for xliff 2.0

Reply:
Added test case for xliff 2.0